autoscale_cloudroast/test_repo/autoscale/system/schedule_policies/test_system_multi_scheduler_webhook_policies.py

```diff
@@ -128,21 +131,22 @@ def test_system_all_types_webhook_and_scheduler_policies_negative(self):
     @tags(speed='quick')
     def test_system_webhook_and_scheduler_policies_many_different_groups(self):
         """
-        Create many groups each with the same type of scheduler and webhook policies and
-        verify the servers after each of their executions
-        """
-        at_style_policy = dict(
-            args=dict(at=self.autoscale_behaviors.get_time_in_utc(30)),
-            cooldown=self.gc_cooldown, type='schedule', name='multi_at_style',
-            change=self.change)
-        group_list = []
-        for each in range(4):
-            group = self._create_multi_policy_group(1, 201, at_style_policy)
-            group_list.append(group.id)
+        Create many groups each with the same type of scheduler and webhook
+        policies and verify the servers after each of their executions
+        """
+        group_ids = []
+        for _ in range(4):
+            policy = dict(
```

Comment:
Non-blocking - might be able to set `self.delta = 30` here, and then just calling `self._create_multi_policy_group(1, 201, self.at_style_policy)`, since you nicely set that to be a property.  :)